scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/DefaultOpEnvironment.java

```diff
@@ -773,7 +766,7 @@ private synchronized void initIdDirectory() {
 			return;
 		}
 		for (String opName : opInfo.names()) {
-			opDirectory.computeIfAbsent(opName, nameListCreator).add(opInfo);
+			opDirectory.computeIfAbsent(opName, name -> new ArrayList<>()).add(opInfo);
```

Comment:
Oh wait no... do we need a field for generic typing reasons?! The lambda won't be inspectable by the ops engine, will it?

Reply:
Or is this not an op so it doesn't matter? Sorry it's late and I'm on mobile, probably I should just scrutinize more carefully myself on a real computer. Anyway, if the lambda generics aren't actually a problem here then I fully support merging this immediately.